src/tls13.c

```diff
@@ -8459,8 +8529,10 @@ static int SendTls13Certificate(WOLFSSL* ssl)
 {
     int    ret = 0;
     word32 certSz, certChainSz, headerSz, listSz, payloadSz;
-    word16 extSz = 0;
+    word16 extSz[1 + MAX_CERT_EXTENSIONS];
```

Comment:
Does this actually need to be `MAX_CERT_EXTENSIONS + 1`? Isn't the +1 already handled by the macro? I reviewed code below and don't see reason. I do see that it needs to be at least 1.

Reply:
I thin we don't need `+ 1` by disabling index increment when not using multi.
```
if (MAX_CERT_EXTENSIONS > extIdx)
   extIdx++;
```